include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -313,7 +313,7 @@ class CloudEnvOptions {
       bool _skip_dbid_verification = false,
       bool _use_aws_transfer_manager = false,
       int _number_objects_listed_in_one_iteration = 5000,
-      bool _constant_sst_file_size_in_sst_file_manager = -1,
+      int _constant_sst_file_size_in_sst_file_manager = -1,
```

Comment:
wow how does this become bool

Reply:
No idea, but I submit the feature doesn't work. 😄  (NB: we haven't used it so far, so can't give you any feedback)